README.md

```diff
@@ -75,41 +97,64 @@ Now, configure the application in the _Settings_ tab.
 1. Configure `Allowed Callback URLs` (comma-separated)
    (The format/suffix of these urls should not be altered.)
 
-   For example, you should include `localhost` and optionally your Netlify app url(s), if you have set up Netlify:
+   You should include `localhost`. Also, you can optionally include your
+   Netlify app url(s), if you have set up Netlify.
+   BCP: What is netlify, and why would I set it up?
+   BCP: Which of these are mandatory for a quick setup? Do the non-mandatory
+   ones hurt anything?  
+   BCP: Is the order critical (aside from the first)?
+   BCP: Should I choose port 3000 for localhost?  
```

Comment:
It's not mandatory, but it's how the documentation is written. If you configure it as a different port everywhere I imagine it would work fine.

Reply:
I don't think we set the Snowpack port number anywhere. So trying to customise it would probably reveal at least one missing config var.